### ISSUE TITLE
Removing focus from URL by closing keyboard causes the news feed to jump to the top of the tab view

### ISSUE THREAD

**Issue by srirambv** [brave/brave-browser#21111]
## Description 
Removing focus from URL by closing keyboard causes the news feed to jump to the top of the tab view

## Steps to reproduce 

   1. Enable News on beta
   2. Open a new tab and set focus to URL bar
   3. Remove focus from URL bar by hiding the keyboard
   4. Brave news jumps to top of the NTP view

## Actual result 

https://user-images.githubusercontent.com/17010094/154112554-c5ebff5f-5e11-4256-8e5c-736c7bbc95f9.mp4

## Expected result
Should only show the feed when user scrolls down the feed

## Issue reproduces how often 
Easy

## Version/Channel Information:

- Can you reproduce this issue with the current Play Store version? NA
- Can you reproduce this issue with the current Play Store Beta version? Yes
- Can you reproduce this issue with the current Play Store Nightly version? Yes

## Device details
   - Install type (ARM, x86): ARM
   - Device type (Phone, Tablet, Phablet): Tablet
   - Android version: 10
   
## Brave version
1.36.94 Beta build

### Website problems only
- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards?NA
- Is the issue reproducible on the latest version of Chrome? NA

### Additional information

cc: @alexsafe

**Comment by kjozwiak** [comment]
@tapanmodh technically this should still be in the `1.39.x` milestone as the original PR landed in `1.39.x` and this wasn't uplifted yet. Once https://github.com/brave/brave-core/pull/12935 is merged into `1.38.x`, the release team will move the issue into the correct milestone, in this case, `1.38.x`.

**Comment by kjozwiak** [comment]
Above requires `1.38.85` or higher for `1.38.x` verification.

**Comment by Uni-verse** [comment]
Verified using `1.38.99, Chromium 100.0.4896.127`

Devices:
`Samsung Galaxy S21` | `Android 12`
`Samsung Galaxy Tab S7` | `Android 12`

- Verified closing keyboard does not cause feed to jump to the top of the tab view.

https://user-images.githubusercontent.com/17885425/164116662-f9eaebaa-d9d8-4e8a-9423-c93db876babc.mp4
